### PR TITLE
app-i18n/transifex-client: add new version 0.14.2

### PR DESCRIPTION
**Pull request by robert7k**

### CODE REVIEW

File: app-i18n/transifex-client/transifex-client-0.14.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (juippis) [inline app-i18n/transifex-client/transifex-client-0.14.2.ebuild]: Can you try to add 3.9?

File: app-i18n/transifex-client/transifex-client-0.14.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="test? ( dev-python/mock[${PYTHON_USEDEP}] )"
+RDEPEND="dev-python/GitPython[${PYTHON_USEDEP}]
+	<dev-python/python-slugify-5.0.0[${PYTHON_USEDEP}]
```
Comment (juippis) [inline app-i18n/transifex-client/transifex-client-0.14.2.ebuild]: Could you verify this...

File: app-i18n/transifex-client/transifex-client-0.14.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	dev-python/requests[${PYTHON_USEDEP}]
+	<dev-python/six-2.0.0[${PYTHON_USEDEP}]
```
Comment (juippis) [inline app-i18n/transifex-client/transifex-client-0.14.2.ebuild]: ... and this, and if upstream file says these are required, could you patch off these restrictions and run python tests?

**Review by juippis — APPROVED** [review]

File: app-i18n/transifex-client/transifex-client-0.14.2.ebuild (hunk 1 above)
Comment (robert7k) [reply app-i18n/transifex-client/transifex-client-0.14.2.ebuild]: Done in #18603

**Pull request closed without merging**